README.md

```diff
@@ -1,24 +1,114 @@
-# README
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# Rent-A-Green
 
-Things you may want to cover:
+> This application allow users to book an appointment to test an electric car. The homepage is a showcase of the cars. You can also add new cars, view the details of a car, and book an appointment for renting a car, among other features.
 
-* Ruby version
+This back-end API is complemented by the [front-end](https://github.com/alexpuente0/final-capstone-frontend)
 
-* System dependencies
+## Built With
 
-* Configuration
+- Ruby on Rails
+- PostgreSQL
+- devise / devise-jwt - for authentication
+- cancancan - for authorization
 
-* Database creation
+### Getting Started
 
-* Database initialization
+To get a local copy up and running follow these simple steps:
 
-* How to run the test suite
+### Prerequisites
 
-* Services (job queues, cache servers, search engines, etc.)
+- Ruby ruby-3.1.2 installed
+- PostgreSQL dbms running
 
-* Deployment instructions
+### Setup
 
-* ...
+- Open your terminal and navigate to the folder you would like to save the project.
+
+- Type these commands into the terminal:
+
+```
+git clone https://github.com/acolombo1/final-capstone-back-end.git
+cd final-capstone-back-end
+bundle install
+```
+
+- You will need a `.env` file before you install and run the project. The `.env` file contains environment variables needed to deploy the webpage. There is a commented `.env.example` file you can use as a guide to configure your own. Type the following commands into the terminal (or Git Bash, if using Windows) to create your `.env` file:
+
+```
+cp .env.example .env
+
+rails secret
+```
+
+- Edit file .env
+
+  - Provide the values for the variables
+    - DATABASE_HOST
+    - DATABASE_USER
+    - DATABASE_PASSWORD
+    - DEVISE_JWT_SECRET_KEY
+  - Assign the value returned by the `rails secret` to DEVISE_JWT_SECRET_KEY.
+
+- Type this commands into the terminal:
+
+```
+rails db:reset
+```
+
+### Run application
+
+- Type this command into the terminal:
+
+```
+rails server
+```
+
+- Open your web browser at [http://127.0.0.1:3000/](http://127.0.0.1:3000/)
+
+### Testing
+
+- Type this command into the terminal:
+
+```
+rspec
+```
+
```

Comment:
- [ ] For some reason, I couldn't run these tests on the terminal:

![image](https://user-images.githubusercontent.com/57421823/192939612-00c7dee2-7fbb-45f8-9fb2-81cda58ef322.png)

If tests are running for you on your environment, you can ignore this comment.  If not, please kindly correct this issue.